tests/warplib.test.ts

```diff
@@ -1,31 +1,23 @@
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
-import { findCairoSourceFilePaths } from '../src/io';
-import { cleanupSync, SafePromise, starknetCompile, wrapPromise } from './util';
+import path from 'path';
+import { cairoTest, SafePromise, wrapPromise } from './util';
 
-const paths = findCairoSourceFilePaths('warplib', true);
-describe('Warplib files should compile', function () {
+describe('Warplib files should compile and execute correctly', function () {
   this.timeout(1800000);
 
-  let compileResults: SafePromise<{ stderr: string }>[];
-
+  let compileResult: SafePromise<{ stderr: string }>;
```

Comment:
Is there a reason not to define it directly within `it`? I don't see any reason to run tests within `before`